src/main/twirl/uk/gov/hmrc/play/views/helpers/dateFieldsFreeInline.scala.html

```diff
@@ -13,7 +13,7 @@
 * See the License for the specific language governing permissions and
 * limitations under the License.
 *@
-@(formItem:Form[_], fieldName: String, args: (Symbol,Any)*)(implicit lang: play.api.i18n.Lang)
+@(formItem:Form[_], fieldName: String, args: (Symbol,Any)*)(implicit messages: Messages)
```

Comment:
Please share the rationale for dropping/replacing `implicit lang` as a parameter.  It can't be implied by twirl version as that is unchanged.  Ta.